sentry_streams/sentry_streams/pipeline/msg_parser.py

```diff
@@ -0,0 +1,15 @@
+from json import JSONDecodeError, dumps, loads
+from typing import Any, Mapping, cast
+
+
+def json_parser(msg: bytes) -> Mapping[str, Any]:
```

Comment:
These are supposed to be the standard serializer/deserializer functions that the user will plug in. Chose them to be functions so that the typing inference works out more seamlessly between steps in a chain, but up to change